src/libertem/udf/base.py

```diff
@@ -919,6 +965,31 @@ def __init__(self, *args, **kwargs) -> None:
         self.params = UDFKwargsWrapper({})
         self.results = UDFData({})
 
+    def _update_parameters(self, new_kwargs):
+        """
+        Update the parameters _for this UDF instance_ to `new_kwargs`.
+
+        For properly supporting AUX data (i.e. `BufferWrapper`-typed parameters),
+        make sure to set the correct views afterwards.
+
+        :meta private:
+        """
+        self.params = UDFKwargsWrapper(new_kwargs)
```

Comment:
See comment above, only point where this is called?

Reply:
Removed.